src/models/hooks/goal.js

```diff
@@ -57,6 +57,22 @@ const preventNameChangeWhenOnApprovedAR = (sequelize, instance) => {
   }
 };
 
+const invalidateSimilarityScores = async (sequelize, instance, options) => {
+  const changed = Array.from(instance.changed());
+
+  if (changed.includes('name')) {
+    await sequelize.models.SimScoreCache.destroy({
+      where: {
+        [sequelize.Op.or]: [
```

Comment:
this errors when you try to change a goal (sequelize.Op is ```undefined```), might need ```const { Op }  = require('sequelize');``` in this file

Reply:
🙏 e02b41a00e0385cc3d788b045cbaf0a9fe0b4750